lib/OpenQA/Script/Client.pm

```diff
@@ -150,6 +150,12 @@ sub run {
         $url->query([%params]) if %params;
     }
 
+    if (defined $params{template} && $params{template} =~ /^file:(.+)$/) {
```

Comment:
The same counts as for the occurrence in `Command.pm`. I'm also wondering the the duplication is necessary here.

Reply:
Figured I needed to extend both `openqa-client` and `openqa-cli` so the option is available in both, and couldn't think of anything better than to duplicate the code. In a sense, `OpenQA::Command::parse_params()` is already a duplicate of code from `OpenQA::Script::Client`. I'll see how it looks after I apply changes with `Mojo::File::slurp` and `Mojo::URL` and make a decision regarding this.

Reply:
Ah, ok. If it is just for the both clients it makes sense. Since it'll be just one line using slurp I suppose it wouldn't make sense to create a separate function.

Reply:
> Ah, ok. If it is just for the both clients it makes sense. Since it'll be just one line using slurp I suppose it wouldn't make sense to create a separate function.

So it's 2 lines inside an if-block in both modules. Should I attempt to factorize this so it's in a single location? I think it'd add more complexity to be honest. Let me know what you think.